cloud/aws/templates/aws_oidc/main.tf

```diff
@@ -29,6 +29,9 @@ resource "aws_db_instance" "civiform" {
     Name = "${var.app_prefix} Civiform Database"
     Type = "Civiform Database"
   }
+
+  # If not null, destroys the current database, replacing it with a new one restored from the provided snapshot
```

Comment:
Do we need this comment here given it's the same as the variable description? I worry they could get out of sync.

Reply:
I find my eyes just slide past it if looking at the whole file with it being a single line comment. You indicated that there's a manual step needing to be done in the AWS console to allow it to run. That might be enough of a safeguard.

Oh and now that I think about it. It's probably unlikely the civic entity doing the deploying will look at the comment in that file.